src/databricks/labs/ucx/source_code/files.py

```diff
@@ -153,15 +163,19 @@ def exists(self, path: Path) -> bool:
     def full_path(self, path: Path) -> Path | None:
         if path.is_file():
             return path
-        for parent in self._syspath_provider.paths:
+        child = Path(self._path_lookup.cwd, path)
```

Comment:
this change is not needed, yield cwd from `paths`.

Reply:
done